### ISSUE TITLE
Screen Reader Unable to Read Bar Numbering Correctly

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#19507]
### Issue type

Other type of issue

### Bug description

When the measure properties is used to change the print numbering the screen reader still only reads them as per the status unchanged.  This means with an anacrusis or other changes the measure numbers being read do not match the place being referred to by a print using student, teacher, colleague, collaborater, etc.  It also has a significant effect on Braille transcription as the print numbers do not export in the xml.  Braille uses a short hand for any bars that are repeated in the score meaning that to fix the numbering to match the print isn't just a case of renumbering the bars correctly but finding all the references to other bars that are repeated and changing them all also, very confusing and complicated.

### Steps to reproduce

With NVDA enabled open any score with an anacrusis where the first full bar is being counted as print bar 1.  right arrow along the notes of the anacrusis into bar 1 and you will hear that the anacrusis is being read as bar 1 and the first full bar as bar 2.  Use the measure property setting to adjust any other print bar numbers and arrow along them to see it has no effect on the screen reader.  

### MuseScore Version

4

### Regression

I don't know

### Operating system

Windows 10

**Comment by bkunda** [comment]
@shoogle can you pls check whether this issue reproduces in Narrator? 
If it does, let's re-prioritise it.

**Comment by jmwmusic** [comment]
This now seems to be fixed in 4.4.3 as the screen reader reads out both actual measure and displayed measure if they differ.  However, see feature request 25403 to get some control over which is read, or both with a toggle.

**Comment by cbjeukendrup** [comment]
Looks like this was indeed implemented in https://github.com/musescore/MuseScore/pull/23332, so I'll close it. Feature request #25403 seems a good idea to me.
